src/components/HomeTab/index.jsx

```diff
@@ -22,13 +21,14 @@ import Row from 'react-bootstrap/Row';
 const STATUS_COLOR_MAP = {
   running: 'rgba(23, 162, 184, 0.7)',
   error: 'rgba(220, 53, 69, 0.7)',
-  success: '#148F68' // invest green
-}
-
-export class HomeTab extends React.PureComponent {
-  /** Renders a button for each invest model and for each cached invest job.
-  */
-
+  success: '#148F68', // invest green

```

Comment:
InVEST Green! Should be a paint color you can find in stores, haha

Reply:
Should probably also be a color we find in the CSS store instead of this jsx module! But I think that's business for another PR.